crates/noirc_evaluator/src/ssa/acir_gen.rs

```diff
@@ -445,37 +542,208 @@ impl Acir {
         }
         sum
     }
+
+    pub fn prepare_inputs(
```

Comment:
Prepare for what? And inputs of what?

Reply:
The inputs of intrinsic functions are transformed into witnesses. I have added a comment.